script_vts.py

```diff
@@ -203,21 +150,90 @@ def handle_file():
     print(f"[+] The file report can be found as {filename}.txt in {Path.cwd()}")
     return 0
 
+##########################################################
+# HELPERS
+##########################################################
 
-def handle_file_report(report_data_dic, string_argument):
-    """Function used to handle our file report"""
-    response = search_api_request(api_key, string_argument)
-    if response is None:  # If we receive nothing quit with error
-        print("[-] We didn't receive something from our search request.")
-        return 1
-    error, report_data_dic, string_parse_json, res_dic, analysis_stat, res_parse_json = parse_json(report_data_dic, response, string_argument)
-    if error is not True:
-        data_process_variable,  report_data_dic = data_processing(string_parse_json, res_dic, analysis_stat, res_parse_json, report_data_dic)
-        if data_process_variable is True:  # check if we need to call for file reputation
-            response_file_reputation = file_reputation_api_request(api_key, string_argument)
-            report_data_dic.append("[+] Response from File reputation request: \n")
-            report_data_dic.append(response_file_reputation)
-    return report_data_dic
+
+def exception_handler(print_exception=False, exception=""):
+    """This function enhances default Python Error handling
+    It will print the line in code that the error occurred on"""
+    if print_exception is True:
+        print(exception)
+        exc_type, exc_obj, exc_tb = sys.exc_info()
+        del exc_type, exc_obj
+        fname = os.path.split(exc_tb.tb_frame.f_code.co_filename)[1]
+        print("Exception on line: ", exc_tb.tb_lineno, " in ", fname)
+    return None
+
+
+def verify_at_least_one_command_line_argument(arguments):
+    """Function checking the number of arguments"""
+    if len(arguments) > 1:
+        return arguments
+    return None
+
+
+def formating_file():
+    """ Checking if the file is well formatted (no blank line, a string on each line),
+    if not format the file"""
+    try:
+        with open(sys.argv[2], encoding="UTF-8") as in_file, open(sys.argv[2], 'r+', encoding="UTF-8") as out_file:
+            out_file.writelines(line for line in in_file if line.strip())
+            out_file.truncate()
+    except IOError as strip_err:
+        exception_handler(True, strip_err)
+        exit(1)
+    return None
+
+
+def get_path(prompt):
```

Comment:
Need to update function name. It should mention that it's accepting user input. Something like:
`get_path_from_user`